checks.d/http_check.py

```diff
@@ -83,7 +84,7 @@ def _check(self, instance):
                 auth = (username, password)
 
             r = requests.get(addr, auth=auth, timeout=timeout, headers=headers,
-                             verify=not disable_ssl_validation)
+                             verify=False if disable_ssl_validation else instance_ca_certs)
```

Comment:
That seems weird, disable_ssl_validation is a boolean while instance_ca_certs is a string or a list of strings